concent_api/common/constants.py

```diff
@@ -60,6 +60,7 @@ class ErrorCode(enum.Enum):
     QUEUE_WRONG_STATE                                                   = 'queue.wrong_state'
     REQUEST_BODY_NOT_EMPTY                                              = 'request_body.not_empty'
     SUBTASK_DUPLICATE_REQUEST                                           = 'subtask.duplicate_request'
+    TRANSACTION_NONCE_NOT_MATCH                                         = 'transaction.nonce.not_match'
```

Comment:
`TRANSACTION_NONCE_DOES_NOT_MATCH` or `TRANSACTION_NONCE_MISMATCH` would be a better name.